code/game/objects/effects/spawners/random/trash.dm

```diff
@@ -139,6 +139,7 @@
 	loot = list( // This spawner will scatter water related items around a moist site.
 		/obj/item/clothing/head/cone = 7,
 		/obj/item/clothing/suit/caution = 3,
+		/mob/living/simple_animal/axolotl = 1,
```

Comment:
These are organized by rarity I think, so axolotl would have to be last as it's the rarest.